ecommerce/serializers.py

```diff
@@ -519,11 +530,26 @@ def validate_items(self, items):
 
             if product_id is None:
                 raise ValidationError("Invalid request")
-            if not models.ProductVersion.objects.filter(
-                product__id=product_id
-            ).exists():
+
+            try:
+                # product_id = product_id
+                product_id = int(product_id)
+                product = models.Product.all_objects.filter(id=product_id).first()
```

Comment:
same question as asked above.

Reply:
It will filter `all_objects` because it's giving validation error for inactive products and also being used in the test for checking invalid products. This line is a part of `validate_item` function.